packages/@okta/vuepress-site/docs/guides/oie-upgrade-add-sdk-to-your-app/main/go/langspecificmapguide.md

```diff
@@ -0,0 +1 @@
+After you’ve completed adding the SDK to your app, the next step is to [get set up](/docs/guides/oie-embedded-common-org-setup/go/main/) with your Okta org. Also, check out the [sample apps](/docs/guides/oie-embedded-common-run-samples) to see working examples of all the supported use cases using the Identity Engine SDK.
```

Comment:
Might be redundant. I think that saying "after you've added" implies completion.
```suggestion
After you add the SDK to your app, the next step is to get [set up](/docs/guides/oie-embedded-common-org-setup/go/main/) with your Okta org. Also, check out the [sample apps](/docs/guides/oie-embedded-common-run-samples) to see working examples of all the supported use cases using the Identity Engine SDK.
```